pravega-cli/src/main/java/io/pravega/tools/pravegacli/commands/controller/ControllerDescribeStreamCommand.java

```diff
@@ -122,18 +131,42 @@ public static CommandDescriptor descriptor() {
     }
 
     private SegmentHelper instantiateSegmentHelper(CuratorFramework zkClient) {
+
+        // hostMonitorEnable should be true for pks clusters
         HostMonitorConfig hostMonitorConfig = HostMonitorConfigImpl.builder()
-                                                                   .hostMonitorEnabled(true)
+                                                                   .hostMonitorEnabled(false)
+                                                                   .hostContainerMap(getHostContainerMap(getCLIControllerConfig().getSegmentStoreURI(),
+                                                                           getServiceConfig().getContainerCount()))
                                                                    .hostMonitorMinRebalanceInterval(Config.CLUSTER_MIN_REBALANCE_INTERVAL)
                                                                    .containerCount(getServiceConfig().getContainerCount())
                                                                    .build();
         HostControllerStore hostStore = HostStoreFactory.createStore(hostMonitorConfig, StoreClientFactory.createZKStoreClient(zkClient));
         ClientConfig clientConfig = ClientConfig.builder()
                                                 .controllerURI(URI.create((getCLIControllerConfig().getControllerGrpcURI())))
-                                                .validateHostName(getCLIControllerConfig().isAuthEnabled())
-                                                .credentials(new DefaultCredentials(getCLIControllerConfig().getPassword(), getCLIControllerConfig().getUserName()))
+                                                .validateHostName(false)
+                                                .credentials(new DefaultCredentials(getCLIControllerConfig().getPassword(),
+                                                        getCLIControllerConfig().getUserName()))
                                                 .build();
         ConnectionPool pool = new ConnectionPoolImpl(clientConfig, new SocketConnectionFactoryImpl(clientConfig));
         return new SegmentHelper(pool, hostStore, pool.getInternalExecutor());
     }
+
+    private static Map<Host, Set<Integer>> getHostContainerMap(List<String> uri, int containerCount) {
```

Comment:
This should be a method implemented in the test case for describe stream, to make it work in the standalone case (i.e., the integration test used).

Reply:
Yes, the changes necessary to make this work in an integration test against the standalone have been made. The method `getHostContainerMap` is only used explicitly for testing.